### PR TITLE
add workaround for old modules

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: config/config.py [hunk 1]
```diff
@@ -55,6 +55,14 @@
     'fosscuda-2020a',
 ]
 
+# workaround for old modules which emit a warning upon load and non-zero exit code
+prepare_cmds = [
+    'shopt -s expand_aliases',
+    "alias module='module -q'",
```
Comment (wpoely86) [inline config/config.py]: why is this needed in general? We only use recent modules for the tests?

Comment (smoors) [reply config/config.py]: indeed, this broke some Lmod tests, so i've put it behind an environment variable.

currently, `prepare_cmds` in the config file is the only place where you can add commands *before* the module loads in the job script.

there are plans to add support for changing those parameters on the cmd line
https://github.com/reframe-hpc/reframe/issues/2675

**Review by wpoely86 — APPROVED** [review]
